tower-grpc-build/src/server.rs

```diff
@@ -387,9 +387,8 @@ macro_rules! try_ready {
 
         let mut poll_trailers_catch_all = codegen::Block::new("Err(ref status) =>");
         poll_trailers_catch_all
-            .line("let mut map = http::HeaderMap::new();")
-            .line("map.insert(\"grpc-status\", status.to_header_value());")
-            .line("Ok(Some(map).into())")
+            .line("let map = status.to_header_map();")
+            .line("Ok(Some(map?).into())")
```

Comment:
This might be clearer as 
```rust
status.to_header_map().map(|map| Some(map).into())
```
instead of using `?` from inside of an `Ok(...)` expression...

Reply:
Done